core/templates/dev/head/components/html_select_directive.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/ng-template" id="components/html_select">
+  <div class="oppia-html-select" >
+    <button class="btn btn-default dropdown-toggle oppia-html-select-selection" type="button" id="menu1" aria-expanded="false" data-toggle="dropdown">
+      <div class="oppia-html-select-selection-element" angular-html-bind="options[ selection ].val"></div>
+      <span class="caret oppia-html-select-selection-caret"></span>
+    </button>
+    <ul class="dropdown-menu" style="width: inherit;">
+      <li ng-repeat="choice in options">
+        <a class="oppia-html-select-option" angular-html-bind="choice.val" ng-click="$parent.selection=choice.id">
```

Comment:
Do you need $parent here? We try to avoid that construct in general, since it has led to issues in the past where additional layers get inserted in between the parent and the child.

Better would be to bind this to a method in the directive controller that sets the selection; would that be possible instead?